.gitignore

```diff
@@ -6,9 +6,11 @@ yarn-error.log
 
 bootstrap/compiled.php
 app/storage/
+resources/docs/
 
 public/storage
 public/hot
+public/docs/
```

Comment:
These files should be ignored and we should run the command that generate the api docs on the live app so it will be live. 
These file are generated whenever we run php artisan apidoc:generate, So it's not reasonable to have them in every commit.

Reply:
We run php artisan migrate:fresh on the live server as well, but we don't ignore the migration files.

Reply:
Migration files are not generated when we run this command, otherwise the docs files are regenerated every time we run it's command.